src/shims/intrinsics.rs

```diff
@@ -449,7 +449,32 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                 let layout = this.layout_of(ty)?;
                 if layout.abi.is_uninhabited() {
                     // Return here because we paniced instead of returning normally from the intrinsic.
-                    return this.start_panic(&format!("Attempted to instantiate uninhabited type {}", ty), unwind);
+                    return this.start_panic(&format!("attempted to instantiate uninhabited type {}", ty), unwind);
```

Comment:
I made this lower case to match the actual error in the rust side which is lower-case https://github.com/rust-lang/rust/blob/303d8aff6092709edd4dbd35b1c88e9aa40bf6d8/src/librustc_codegen_ssa/mir/block.rs#L478